src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig

```diff
@@ -21,7 +21,12 @@
                     {{ log.channel }}
                 </td>
             {% endif %}
-            <td>{{ log.message|format_log_message(log.context) }}</td>
+            <td>
+                {{ log.message|format_log_message(log.context) }}
+                {% if log.context ?? false %}
+                    <span class="block text-muted prewrap m-t-5">{{ log.context|json_encode(constant('JSON_PRETTY_PRINT') b-or constant('JSON_UNESCAPED_UNICODE') b-or constant('JSON_UNESCAPED_SLASHES')) }}</span>
```

Comment:
it is is a block display with `prewrap`, why not using `<pre>` ? It would be the dedicated markup rather than a `span`